usaon_vta_survey/models/tables.py

```diff
@@ -133,11 +133,11 @@ class Survey(BaseModel):
         nullable=False,
     )
 
-    # Do we want this to be a dropdown?
-    status = Column(
-        String(128),
+    status_id = Column(
+        String,
+        ForeignKey('status.id'),
+        default="work in progress",
```

Comment:
If we eventually change the status values (e.g. from `work in progress` -> `wip`) we'd need to remember to change this default string. Using `STATUSES[0]` would solve this problem, but would also create an implicit dependency on the order of the `STATUSES` list being correct. We could put a comment next to the constant list saying that the first element is the default so people editing the list have the context we need. We may also want to type `STATUSES` as `Final[list[str]]`.

Reply:
Great idea.